packages/cli/package.json

```diff
@@ -45,6 +45,8 @@
     "@basemaps/sprites": "^6.29.0",
     "@chunkd/fs": "^8.4.0",
     "@cogeotiff/core": "^7.0.0",
+    "@cotar/core": "^5.4.0",
```

Comment:
can you check to see if we are using another version of cotar somewhere else? the yarn lock changes seem excessive!

Reply:
Fixed.